README.rst

```diff
@@ -330,160 +330,3 @@ Credits
   `PyPI`_.
 - `Paul Kehrer <https://github.com/reaperhulk>`_ for his
   `awesome blog post <https://langui.sh/2016/12/09/data-driven-decisions>`_.
-
-Changelog
```

Comment:
Shall we add a link to the new changeling file? It could be a plain link or a badge.

Reply:
I added:

```toml
[project.urls]
Changelog = "https://github.com/ofek/pypinfo/blob/master/CHANGELOG.rst"
```

which should add a button on https://pypi.org/project/pypinfo/. Should I also add a link here?

Reply:
I saw, that's really good :)

That should be fine, we can always add it later.